README.md

```diff
@@ -35,7 +42,7 @@ pip install kedro-mlflow
 ```
 If you want to use the ``develop`` version of the package which is the most up to date, you can install the package from github:
 ```console
-pip install --upgrade git+https://github.com/quantumblacklabs/kedro.git@develop
+pip install --upgrade git+https://github.com/Galileo-Galilei/kedro-mlflow.git@develop
```

Comment:
Haha, I didn't notice this mistake before!

Reply:
Me neither, I don't know ow it survives that long!